docs/operations/kube_apiserver_loadbalancing.md

```diff
@@ -42,6 +43,15 @@ server authenticates requests by itself. It is used for the token based authenti
 the mTLS connection is set by the lua script mentioned above only. This is a safety net to prevent that requests can
 reach Kube API server via the trusted connection in case the lua scripts fails for some reason.
 
+Cluster internal control plane components like `kube-controller-manager`, `kube-scheduler` and `gardener-resource-manager`
+use L7 load balancing too. They connect to the Kube API server via an cluster IP service for istio ingress gateway.
+The generic token kubeconfig uses the public Kube API server endpoint. In order to avoid external traffic, the control
+plane components use host aliases in their pod specifications. For convenience, the host alias are automatically added
+by the [`pod-kube-apiserver-load-balancing`](../../pkg/resourcemanager/webhook/podkubeapiserverloadbalancing) webhook
+in `gardener-resource-manager`. It also adds a label to create a network policy allowing egress traffic to the istio
```

Comment:
Do you also want to add documentation about the webhook in https://github.com/gardener/gardener/blob/master/docs/concepts/resource-manager.md#webhooks?